internal/compiler/widgets/fluent-base/widget-table-view.slint

```diff
@@ -0,0 +1,179 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+import { LineEditInner } from "../common/common.slint";
+import { Palette, ScrollView } from "std-widgets-impl.slint";
+
+component TableViewColumn inherits Rectangle {
+    callback clicked <=> touch-area.clicked;
+
+    in property <SortOrder> sort-order: SortOrder.unsorted;
+    
+    touch-area := TouchArea {}
+
+    HorizontalLayout {  
+        padding-left: 16px;
+        padding-right: 16px;
+        spacing: 2px;
+
+        HorizontalLayout { 
+            @children
+        }
+
+        Path {
+            opacity: root.sort-order != SortOrder.unsorted ? 1.0 : 0.0;
```

Comment:
Would it perhaps make sense here to use visible instead of opacity? @ogoffart What do you think?

Reply:
makes sense and is changed.